### ISSUE TITLE
Default `help` and `completion` command descriptions are inconsistent

### ISSUE THREAD

**Issue by brianstrauch** [spf13/cobra#1458]
As of v1.2.0, there are two commands that are included by default in every Cobra CLI. Their formatting is inconsistent, which is incredibly sloppy, especially considering that their descriptions can't be changed; to my knowledge these commands can only be hidden.

First, one `Short` description is uppercased, and one is lowercased. Recommendation: Make them both uppercase.
```
Available Commands:
  completion  generate the autocompletion script for the specified shell
  help        Help about any command
```

Second, one `Long` description has an extra leading newline, and the other does not. Recommendation: Remove the extra newline.
```
$ cmd completion -h

Generate the autocompletion script for spotify for the specified shell.
See each sub-command's help for details on how to use the generated script.

...
```

```
$ cmd help -h
Help provides help for any command in the application.
Simply type cmd help [path to command] for full details.

...
```

In conclusion, it looks like the new `completion` command did not even attempt to maintain consistent formatting. A good long term solution would be to allow users to edit these descriptions to match their preferred style.

**Comment by marckhouzam** [comment]
This is my fault. The inconsistency comes from the fact that I based the command on helm's completion command which formats it's command texts differently.

1- #1455 fixes the capitalization
2- for the extra empty line, it would be worth suggesting it in #1455  or posting another PR
3- I was thinking it may be a good idea to make the default completion commands easily accessible so that some of its fields could be overridden

**Comment by jxsl13** [comment]
is it possible to remove those parenthesis around command completion descriptions or replace them with your own description prefix/suffix?

**Comment by marckhouzam** [comment]
> is it possible to remove those parenthesis around command completion descriptions or replace them with your own description prefix/suffix?

Currently no, but it wouldn't be difficulty to make the format configurable for bash v2 and PowerShell.  Note that for zsh and fish the description format is handled by the shell.

**Comment by invidian** [comment]
Not stale as we have #1455.
